spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloAttributeStore.scala

```diff
@@ -41,6 +41,19 @@ class AccumuloAttributeStore(connector: Connector, val attributeTable: String) e
     scanner.iterator.map(_.getValue)
   }
 
+  private def _delete(layerId: LayerId, attributeName: Option[String]): Unit = {
```

Comment:
Just wanted to keep code style: in the `S3AttributeStore` we need `_delete` function

```scala
private def _delete(layerId: LayerId, path: String): Unit = {
  if(!layerExists(layerId)) throw new LayerNotFoundError(layerId)
  s3Client.deleteObject(bucket, path)
}

def delete(layerId: LayerId): Unit =
  _delete(layerId, attributePath(layerId))

  def delete(layerId: LayerId, attributeName: String): Unit =
    _delete(layerId, attributePath(layerId, attributeName))
```